cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -273,6 +273,7 @@ globalCommand commands = CommandUI {
         , addCmd "hscolour"
         , addCmd "exec"
         , addCmd "list-bin"
+        , addCmd "ide"
```

Comment:
Same as above, I think you meant "status" here. :)

Reply:
Why did I even add it here...

Does this even do anything? Someone needs to clean up the cli. (Or document it)

Reply:
ah, I see, it does make sense and is required. Still :D